source/server-discovery-and-monitoring/tests/rs/use_setversion_without_electionid.yml

```diff
@@ -97,21 +97,21 @@ phases: [
         outcome: {
             servers: {
                 "a:27017": {
+                    type: "RSPrimary",
+                    setName: "rs",
+                    setVersion: 1
```

Comment:
Is this intentionally missing electionId? I think it should be:
```
electionId: {"$oid": "000000000000000000000002"},
```

Reply:
Yep that should be there, added!